src/client/vite.config.ts

```diff
@@ -156,38 +153,23 @@ const customPreloadPlugin = () => {
   return result
 }
 
-const eslintPlugin = {
```

Comment:
as you have removed these, do we need to add the relevant steps into the build yaml?

Reply:
yes, planning to do that in a separate PR. as it is the current implementation was not preventing errors from being committed.